code/modules/clothing/spacesuits/ert_hardsuits.dm

```diff
@@ -76,29 +76,29 @@
 //Commander
 /obj/item/clothing/head/helmet/space/hardsuit/ert/commander
 	name = "emergency response team commander helmet"
-	desc = "A helmet worn by the commander of a Nanotrasen Emergency Response Team. Has blue highlights. Armoured and space ready."
+	desc = "An environmentally sealed combat helmet with a wide plexiglass visor for maximum visibility. This one's got blue Command stripes."
```

Comment:
```suggestion
	desc = "An environmentally sealed combat helmet with a wide plexiglass visor for maximum visibility. This one has blue Command stripes."
```

One's is a contraction for "one is". The possessive of "one" doesn't have an apostrophe.